recipes/yaqd-labjack/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "yaqd-labjack" %}
+{% set version = "2022.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yaqd-labjack-{{ version }}.tar.gz
+  sha256: 01efa7f1d291a6c8842043bf830266dfdf539dc4ffd6924a8e412e5a5156bdb2
+
+build:
+  entry_points:
+    - yaqd-labjack-sensor = yaqd_labjack._labjack_sensor:LabjackSensor.main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - flit
+    - pip
+    - python >=3.7
+  run:
+    - flit
```

Comment:
Why is flit required at runtime?

Reply:
I _think_ it used to be, but recently I've learned that it's not. You're correct!